### DISCUSSION TITLE
Support for multiple versions

### DISCUSSION THREAD

**Discussion by ajayvaaka** [HewlettPackard/POSH-HPOneView#503] [Q&A] [answered]
What is the supported Module for OV 4.20.xxx and OV 5.20.xxxx ? . We are under the process of migration from 4.x to 5.x, I'm looking for a module that supports both version. I tried all the modules, no-luck. I tried to install and remove as per the OV version, it is working sometimes only. Any help, please.

**Comment by ChrisLynchHPE** [comment, marked as answer]
Older library versions do support newer appliance versions.  So, you could use the OneView 4.20 PowerShell library to manage a OneView 5.xx appliance.  But the features of OneView are locked to the version of the library used.  This is how our API behaves.

This is documented [here](https://hpe-docs.gitbook.io/posh-hpeoneview/#table-1-requirements).
